src/uie/src/uie_executor/executor.py

```diff
@@ -0,0 +1,32 @@
+from jina import Executor, requests, DocumentArray, Document, Deployment
+from paddlenlp import Taskflow
+from typing import List, Union, Dict
+
+Schema = Union[
+    # ['时间', '赛手', '赛事名称', '情感倾向[正向，负向]']  实体抽取 分类任务需要[]来设置分类的label
+    str,
+    # {'歌曲名称': ['歌手', '所属专辑']} 实体属性抽取
+    Dict[str, List[str]]
+]
+
+
+class UIEExecutor(Executor):
+    def __init__(self,  **kwargs):
+        super().__init__(**kwargs)
+        self.uie = Taskflow('information_extraction',
+                            schema=['时间', '选手', '赛事名称'])
+
+    @requests
+    async def call_uie(self, docs, parameters: Dict[str, List[Schema]], **kwargs):
+        if (parameters["schema"] != None):
+            self.uie.set_schema(parameters["schema"])
+
+        for d in docs:
+            d.text = str(self.uie(d.text))
```

Comment:
Should we have a structured response? For example, to return the result in `.tags`:
```
result_doc.tags['时间'] = ...
result_doc.tags['选手'] = ...
...
```